crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
+        if value == value:
+            return key
+
+    return "BTC"
+
+@is_command("Check all the crypto prices.")
+def crypto(message_dict):
+    user = message_dict["display-name"].lower()
+    message = [message_dict["message"]]
+    message_dict["suppress_log"] = True
+    
+    # Attempt to split the message and get each item into an array
+    try:
+        message = re.split(', |; |\s', message)
+    # Fail gracefully if it's a single or empty message
+    except Exception as ex:
+        log(f"Exception in crypto.crypto: {str(ex)}")
+
+    # Check each item requested and make sure it's the short-code and not the full name
+    # Convert to short-code from name if needed
+    for item in message:
+        if item.length > 3:
```

Comment:
* Why the length filter? 
* Some codes are shorter than 3 characters - I found 29 codes in my list is with len 2. 
* Although functionally identical, I prefer `len(item)` to match the rest of my code.

Reply:
The length filter is to check if it's a name or short-code, but I probably should improve that logic by doing something more along the lines of `if crypto_data[item]` which would be better logic and also let me quickly handle failure cases rather than running the reverse lookup too.

Reply:
The length would only be a valid check if all of the codes were shorter than all of the names, which is not the case: the longest code I found is 7 characters, and the shortest name is 2. 

I think allowing either a code or a name is overcomplicating it - it should allow only a code. That way it's far simpler. And in practise I don't think people are going to type out the full name, especially if they have the option of using the code.